src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,34 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+          <UserInfo user={user} />
```

Comment:
your user could be potentially `undefined` since we're looking for user using `find` method (it returns undefined if nothing was found)
 please consider this option while rendering  `UserInfo`.

Reply:
Great! We've added a check when we're looking for user

Let's use conditional rendering for `UserInfo` in case user is `null`